crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -82,7 +80,7 @@ impl SimpleFaucet {
         let metrics = FaucetMetrics::new(prometheus_registry);
 
         let wal = WriteAheadLog::open(wal_path);
-        let mut pending = vec![];
+        let mut pending: Vec<(Uuid, SuiAddress, Vec<ObjectID>, TransactionData)> = vec![];
```

Comment:
nit: it would be more readable if you can define a struct for this

Reply:
Yes please -- much easier to follow than looking at `.0`, `.1` etc accesses below.